packages/core/content-manager/shared/contracts/relations.ts

```diff
@@ -15,31 +15,39 @@ export interface Pagination {
   total: number;
 }
 
+type RelationResponse =
+  | {
+      results: RelationResult[];
+      pagination: Pagination;
+      error?: never;
+    }
+  | {
+      results?: never;
+      pagination?: never;
+      error: errors.ApplicationError | errors.YupValidationError;
+    };
+
 /**
  * GET /relations/:model/:targetField
  */
 export declare namespace FindAvailable {
   export interface Request {
     body: {};
-    query: Partial<Pick<Pagination, 'pageSize' | 'page'>> & { _q?: string; _filter?: string };
+    query: Partial<Pick<Pagination, 'pageSize' | 'page'>> & {
+      id: string;
+      _q?: string;
+      _filter?: string;
+      locale?: string | null;
```

Comment:
now that I see this, wdyt if we create a `Documents.Params.Locale` ? that way we could reuse it everywhere else